Description: Avoid linking against libraries which are not used at all
Author: Andreas Tille <tille@debian.org>
Date: Thu, 24 Feb 2011 08:14:04 +0100

Index: autodock-vina-1.1.2/build/makefile_common
===================================================================
--- autodock-vina-1.1.2.orig/build/makefile_common	2011-05-11 22:35:13.000000000 +0200
+++ autodock-vina-1.1.2/build/makefile_common	2011-05-12 13:04:32.960202842 +0200
@@ -10,7 +10,7 @@
 
 LDFLAGS = -L$(BASE)/lib -L.
 
-LIBS = -l boost_system${BOOST_LIB_VERSION} -l boost_thread${BOOST_LIB_VERSION} -l boost_serialization${BOOST_LIB_VERSION} -l boost_filesystem${BOOST_LIB_VERSION} -l boost_program_options${BOOST_LIB_VERSION}#-l pthread
+LIBS = -l boost_system${BOOST_LIB_VERSION} -l boost_thread${BOOST_LIB_VERSION} -l boost_program_options${BOOST_LIB_VERSION}#-l pthread
 
 .SUFFIXES: .cpp .o
 
